templates/zerver/emails/missed_message.source.html

```diff
@@ -63,5 +65,5 @@
 {% endblock %}
 
 {% block manage_preferences %}
-<p><a href="{{ realm_uri }}/#settings/notifications">Manage email preferences</a> | <a href="{{ unsubscribe_link }}">Unsubscribe from missed message emails</a></p>
+<p><a href="{{ realm_uri }}/#settings/notifications">Manage email preferences</a> | <a href="{{ unsubscribe_link }}">Unsubscribe</a> from missed message emails</p>
```

Comment:
Here I have just changed `[Unsubscribe from missed emails]` to `[Unsubscribe] from missed emails`